great_expectations/data_context/util.py

```diff
@@ -93,7 +93,7 @@ def instantiate_class_from_config(config, runtime_config, config_defaults=None):
 
     class_name = config.pop("class_name", None)
     if class_name is None:
-        logger.warning("Instantiating class from config without an explicit class_name is dangerous. Consider adding "
+        logger.debug("Instantiating class from config without an explicit class_name is dangerous. Consider adding "
```

Comment:
A separate PR now adds the class_name default

```suggestion
        logger.warning("Instantiating class from config without an explicit class_name is dangerous. Consider adding "
```